cmd/tools/sql/main.go

```diff
@@ -23,6 +23,7 @@ package main
 import (
 	"os"
 
+	//_ "github.com/uber/cadence/common/persistence/sql/sqlplugin/db2"      // needed to load db2 plugin
```

Comment:
Please don't check in commented out code.

Reply:
the hope is `docs/persistence.md` is read first before trying to run on db2